### DISCUSSION TITLE
Distributed  security onion 2.3.50

### DISCUSSION THREAD

**Discussion by zidan9873** [Security-Onion-Solutions/securityonion#4084] [Unsupported Versions]
Hello Support,
How do you recommended install Distributed  security onion 2.3.50?
The information in the link
https://docs.securityonion.net/en/2.3/architecture.html
Do I need to install 3 servers with Distributed installation?

**Comment by weslambert** [comment]
For a distributed installation, you would need at least one of the following:

(1) Manager
(1) Search Node
(1) Sensor

So, yes, you would need to install Security Onion on (3) servers.  We typically recommend the manager as a VM, but the others are recommended to be physical where possible.
